src/notation/qml/MuseScore/NotationScene/internal/NotationSwitchScrollArrowButton.qml

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2022 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.15
+
+import MuseScore.UiComponents 1.0
+
+FlatButton {
+    id: root
+
+    width: 22
+    height: parent.height
+
+    opacity: 1.0
+
+    backgroundItem: Rectangle {
```

Comment:
Please explain why you redefine the background item?

Reply:
I did it because we don't want the rounded corners in this case, _and_ we want the background a different color, _and_ we want the normal state opacity to be 1.0 instead of ui.theme.buttonOpacityNormal. 
Of course, we can add more properties to FlatButton to control the corner radius and the base state opacity, but I decided not to do that because FlatButton already has so many properties and I don't wanted to complicate it even further.